src/gpg/key/gen.py

```diff
@@ -4,23 +4,23 @@
 
 
 def gen(parameters):
-    params = """<GnupgKeyParms format="internal">
-    {parameters}
-    </GnupgKeyParms>
-    """.format(parameters=parameters)
+    try:
+        params = """<GnupgKeyParms format="internal">
+        {parameters}
+        </GnupgKeyParms>
+        """.format(parameters=parameters)
```

Comment:
Try block should start from line 13 imo i.e Piece of code that are likely to cause exceptions.

Reply:
Yup but it doesn't change the behavior if we have correct lines in try also am am catching specific exceptions now.